allow to set identity_func by ANALYTICAL_IDENTITY_FUNC settings (#189)
jazzband/django-analytical: docs/settings.rst

    ``GOOGLE_ANALYTICS_INTERNAL_IPS`` to configure for Google Analytics.

    See :ref:`internal-ips`.

.. data:: ANALYTICAL_IDENTITY_FUNC

>     Default: Identity function dependend on provider

Reviewer: ```suggestion
    Default: Identity function dependent on provider
```

    A function that returns the identity of the given user. This overrides the

>     default settings of diferent providers.

Reviewer: ```suggestion
    default settings of different providers.
```

>     E.g. Google has in it's conditions for enabling UserID requirement, that prohibits

Reviewer: ```suggestion
    E.g. Google has in its conditions for enabling UserID the requirement, that prohibits
```

>     sending personal data (such as e-mail address) to analytics.

Reviewer: ```suggestion
    sending personal data (such as an e-mail address) to analytics.
```

>     If e-mail address is used as username, using GTag would break the requirements.

Author: I meant transition from `google_analytics` to `google_analytics_gtag` with user identity which I underwent in my project.

The requirements are their terms of use, which they write very suggestively: https://developers.google.com/analytics/solutions/crm-integration#user_id
And also I think I had to agree to that when switching on the User ID feature in Google Analytics.
